client/app/components/faq/faqlist.js

```diff
@@ -23,7 +18,7 @@ const API_URL = `${ADDRESS}:9090/faq`;
 
 
 class FaqList extends Component {
-	_keyExtractor = (item, index) => item.id;
+	static keyExtractor = (item) => item.id;
```

Comment:
Why do we need this `static` keyword?

Reply:
Yeah, so usually I implement static functions for utility functions inside a class so I kinda did it by habit. Open to changes though, what do you think?

Reply:
Sure, that's good practice!  I should pick it up too  😄